test/SlowTests/Issues/RavenDB_14235.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_14235 : RavenTestBase
+    {
+        private readonly List<int> _listOfNumsThatAreNulls = new List<int> { 9, 10, 11, 12, 13 };
+
+        public RavenDB_14235(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(2)]
+        [InlineData(4)]
+        [InlineData(8)]
+        [InlineData(16)]
+        [InlineData(32)]
+        [InlineData(64)]
+        [InlineData(128)]
+        [InlineData(256)]
+        public async Task PassingOnlyEscapedCharactersAsId(int size)
+        {
+            char[] chars = new char[size];
+
+            for (int c = 0; c < 32; c++)
+            {
+                if (c.In(_listOfNumsThatAreNulls))
```

Comment:
I don't understand what this is for.
Did you mean, known special chars here?

Reply:
those are whitespaces, so we save a guid as an ID in place of those